java/azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/ExceptionReceivedEventArgs.java

```diff
@@ -14,12 +14,16 @@
 	private final String hostname;
 	private final Exception exception;
 	private final String action;
+	private final String partitionId;
 	
-	ExceptionReceivedEventArgs(String hostname, Exception exception, String action)
+	public static final String NO_ASSOCIATED_PARTITION = "";
+	
+	ExceptionReceivedEventArgs(String hostname, Exception exception, String action, String partitionId)
 	{
 		this.hostname = hostname;
 		this.exception = exception;
 		this.action = action;
+		this.partitionId = partitionId;
```

Comment:
>partitionId [](start = 21, length = 11)

This can be anything (including null), so technically your assumption that this is either partition id or NO_ASSOCIATED_PARTITION is not really correct. The constructor should default it to NO_ASSOCIATED_PARTITION and only assign if partitionId is a valid string.